Makefile

```diff
@@ -18,20 +18,18 @@ generate:
 	cp plugins.json data/static/
 	go generate ./...
 
-## Runs govet and gofmt against all packages.
+## Runs go vet and golangci-lint against all packages.
```

Comment:
Keeping `go vet` around for sane errors when the code doesn't pass syntax tests. golangci-lint obscures this behind messages about being unable to run a particular checker.

Reply:
Maybe we run `make test` and `make check-style` in parallel? This way `make test` will fail, if there is a syntax error.

> golangci-lint obscures this behind messages about being unable to run a particular checker.

I've not encountered this. Would you please provide an example of an error that gets hidden?

Reply:
I was ambiguous here, sorry. I was talking about running them in parallel in circleci, not in the Makefile.

Reply:
Apologies for misunderstanding -- what's the benefit to running them in parallel on CircleCI? I don't want my tests to fail because of syntax errors /and/ my lint to fail for the same reason. 

Plus, at least for a repository of this size, running them serially is "faster" since we only need to wait for a single CircleCI process, that only has to restore/save cache once.

Reply:
Good point, I'm over engineering.